code/game/atoms.dm

```diff
@@ -160,10 +160,9 @@
 
 /atom/proc/attack_hulk(mob/living/carbon/human/user, does_attack_animation = FALSE)

```

Comment:
is `does_attack_animation` used anywhere anymore, then?

Reply:
No it doesn't seem like it, I'll remove it from the arguments